recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -94,6 +124,26 @@ tdx_enc_partition_mount() {
     fi
 }
 
+# restore data if available
+tdx_enc_restore_data() {
+    if [ ${TDX_ENC_PRESERVE_DATA} -eq 1 ]; then
+        tdx_enc_log "Checking backup file: ${TDX_ENC_PRESERVE_LOCATION}"
+
+        if [ -f ${TDX_ENC_PRESERVE_LOCATION} ]; then
+            tdx_enc_log "Restoring original content..."
+
+            msgs="$(bzip2 -cd ${TDX_ENC_PRESERVE_LOCATION} | tar -C ${TDX_ENC_STORAGE_MOUNTPOINT} -xf -)"
```

Comment:
Is `bzip2` and `tar` part of BusyBox or they are provided by individual packages? If they are part of standalone packages, then you should add them to the list of runtime dependencies in the recipe

Reply:
yes they are. same as awk and grep for oe-core meta layers.